WalletWasabi.Gui/UiConfig.cs

```diff
@@ -24,7 +24,7 @@ public class UiConfig : ConfigBase
 	{
 		private bool _lurkingWifeMode;
 		private bool _lockScreenActive;
-		private string _lockScreenPinHash;
+		private string _lockScreenPinHash = "";
```

Comment:
Just to make sure: This pin hash is supposed to be an empty string at the beginning in some GUI part, right?

Reply:
Yes.